nanomq/apps/broker.c

```diff
@@ -544,7 +544,9 @@ server_cb(void *arg)
 			smsg = NULL;
 			work->msg = NULL;
 			// free conn_param due to clone in protocol layer
-			conn_param_free(work->cparam);
+			if (work->proto != PROTO_MQTT_BRIDGE) {
```

Comment:
@lee-emqx 
Please help me to verify the function of webhook. I am not sure if it will affect the process of webhook

Reply:
I am not familiar with webhook.